src/components/pages/search.js

```diff
@@ -194,6 +200,7 @@ function Search({ PageTemplate }) {
           className="SearchPage-table"
           dataSource={[...displayedRows]}
           rowClassName={() => 'SearchPage-tableRow'}
+          onChange={handleResultTableChange}
```

Comment:
Like discussed, this feature would be super nice to have on the tables from the Redaction page too.

Reply:
Now also the redaction page scrolls!

Reply:
Scrolling is perfection 👌